configure.ac

```diff
@@ -139,7 +139,8 @@ else
     MESOS_CPPFLAGS+=" -I${mesos_root}/3rdparty/libprocess/include"
 
     # Add stout to include path.
-    MESOS_CPPFLAGS="${MESOS_CPPFLAGS} -I${mesos_root}/3rdparty/stout/include"
+    MESOS_CPPFLAGS="${MESOS_CPPFLAGS} -isystem ${mesos_root}/3rdparty/stout/include"
```

Comment:
@GoelDeepak is it possible to make it specific to `boost`. I think that is where the error was originating from. This is what was done for as well:
https://issues.apache.org/jira/browse/MESOS-3799